recipes/tomocupy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "1.0" %}
+
+package:
+  name: tomocupy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomography/tomocupy.git
+  git_rev: develop 
+
+build:
+  # skip: True  # [cuda_compiler_version != "11.0"]
+  # skip: True  # [py != 310]
+  # skips above this line are for debugging purposes only
+  skip: True  # [cuda_compiler_version in (undefined, "None")]
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - cmake
+    - ninja
+    - swig
+  host:
+    - cudatoolkit  # [cuda_compiler_version != "None"]
+    - numpy
+    - pip
+    - python
+    - scikit-build
+    - setuptools
+  run:
+    - python
+    - cupy
+    - h5py
+    - numexpr
+    - numpy
+    - opencv
+    - pywavelets
+    - tifffile
+
+test:
+   requires:
+     - pip
+   imports:
+     - tomocupy
```

Comment:
This import test might fail because the docker container doesn't have a GPU available. If so, you can comment it out.